nss-day-cohort-60/rare-client-tenacious-tators
# Evn styling

# Description of Proposed Changes
Category and Tag Manager Styling. Some "Dark Mode" styling added
---


# Steps to Test

1. Fetch the `EVN-styling` branch and switch to it
2. Be sure you've pulled the new server PR and seeded your database
3. Check Category Manager and Tag Manager tabs, including the edit and delete (re-seed db after delete)
